core/server/master/src/main/java/alluxio/master/file/FileSystemMasterClientServiceHandler.java

```diff
@@ -300,6 +266,18 @@ public String toString() {
   }
 
   @Override
+  public GetMountTableTResponse getMountTable() throws AlluxioTException {
```

Comment:
👍 
we need this method to link Thrift service API to the real impl in `DefaultFileSystemMaster`